From 2a7dd4555f1d6edfff83fa6057ac496ffe4a0bb9 Mon Sep 17 00:00:00 2001
From: Filipe Brandao <fdabrandao@gmail.com>
Date: Sat, 17 Sep 2016 19:39:30 -0400
Subject: [PATCH] Remove unused and nonportable include

---
 spmatrix/test.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/spmatrix/test.c b/spmatrix/test.c
index 64cc762..d39f267 100644
--- a/spmatrix/test.c
+++ b/spmatrix/test.c
@@ -20,7 +20,6 @@
 #include <config.h>
 #include <stdlib.h>
 #include <math.h>
-#include <unistd.h>
 
 #include <gsl/gsl_math.h>
 #include <gsl/gsl_errno.h>
-- 
2.10.0

