From fe3d6e2393139931a9d77d03c00972c63d2adb51 Mon Sep 17 00:00:00 2001
From: vitaut <victor.zverovich@gmail.com>
Date: Tue, 17 Nov 2015 10:03:45 -0800
Subject: [PATCH] Use "static inline" instead of "inline" to prevent link
 errors in MSVC

---
 gsl_inline.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gsl_inline.h b/gsl_inline.h
index 2172201..8a00378 100644
--- a/gsl_inline.h
+++ b/gsl_inline.h
@@ -49,7 +49,7 @@
 #ifdef HAVE_INLINE
 #  if defined(__GNUC_STDC_INLINE__) || defined(GSL_C99_INLINE) || defined(HAVE_C99_INLINE)
 #    define INLINE_DECL inline  /* use C99 inline */
-#    define INLINE_FUN inline
+#    define INLINE_FUN static inline
 #  else
 #    define INLINE_DECL         /* use GNU extern inline */
 #    define INLINE_FUN extern inline
-- 
2.10.0

