From c7211a268cc91b42dc1d573a9b1804f85c069f37 Mon Sep 17 00:00:00 2001
From: Victor Zverovich <victor.zverovich@gmail.com>
Date: Fri, 19 Aug 2016 11:56:19 -0700
Subject: [PATCH] Fix assertion failure on amd64 when compiled with MSVC (#7)

---
 ieee-utils/fp-win.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ieee-utils/fp-win.c b/ieee-utils/fp-win.c
index fc1c874..e024eae 100644
--- a/ieee-utils/fp-win.c
+++ b/ieee-utils/fp-win.c
@@ -37,7 +37,9 @@ gsl_ieee_set_mode (int precision, int rounding, int exception_mask)
     case GSL_IEEE_DOUBLE_PRECISION:
     default:							mode |= _PC_53;
 	}
+#ifndef _M_AMD64
 	mask |= _MCW_PC;
+#endif
 
 	switch(rounding)
     {
-- 
2.10.0

