From f82576d2ba7e2c915c5694935a7cb9137e0670a5 Mon Sep 17 00:00:00 2001
From: vitaut <victor.zverovich@gmail.com>
Date: Tue, 17 Nov 2015 10:16:47 -0800
Subject: [PATCH] Remove unused and nonportable includes

---
 matrix/test.c | 1 -
 vector/test.c | 1 -
 2 files changed, 2 deletions(-)

diff --git a/matrix/test.c b/matrix/test.c
index 9539ad6..fa3b82e 100644
--- a/matrix/test.c
+++ b/matrix/test.c
@@ -30,7 +30,6 @@
 
 #include <stdlib.h>
 #include <stdio.h>
-#include <unistd.h>
 #include <math.h>
 #include <gsl/gsl_math.h>
 #include <gsl/gsl_matrix.h>
diff --git a/vector/test.c b/vector/test.c
index 649ba9f..51e326d 100644
--- a/vector/test.c
+++ b/vector/test.c
@@ -30,7 +30,6 @@
 #endif
 
 #include <stdlib.h>
-#include <unistd.h>
 #include <fcntl.h>
 #include <stdio.h>
 #include <gsl/gsl_math.h>
-- 
2.10.0

